### DISCUSSION TITLE
auto backup to google drive

### DISCUSSION THREAD

**Discussion by sagar3506** [outline/outline#7480] [Ideas]
please make feature which auto backup data daily on google drive for safety of notes if server crash or anything else bad happen

**Comment by Limezy** [comment]
You can already do it with a small script using the API backup endpoint

**Comment by sagar3506** [comment]
https://your-outline-server/api/backup ?? this way

**Reply by sagar3506** [reply]
not work on my server its give this output i try other endpoints too same with all  {"ok":false,"error":"not_found","status":404,"message":"Resource not found"}

**Reply by Limezy** [reply]
https://www.getoutline.com/developers#tag/collections/POST/collections.export_all
This one maybe ?

**Reply by sagar3506** [reply]
i think something issue at my server what ever i do its give me only 401 error
